<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 25px; 
    background-color: #ffa500; 
    color: white;
    font-weight: bold; 
    border: 2px solid #cc8400; 
}
QPushButton:hover {
    background-color: #cc8400; 
}
QPushButton:pressed {
    background-color: #ff7f00;
}
</string>
    </property>
    <property name="text">
     <string>Wallet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #28a745;
    color: white;
    font-weight: bold; 
    border: 2px solid #218838; 
}
QPushButton:hover {
    background-color: #218838;
}
QPushButton:pressed {
    background-color: #1e7e34; 
}
</string>
    </property>
    <property name="text">
     <string>Food</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 10pt;     
    text-align: center;   
    color: black;        
}
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 10pt;     
    text-align: center;   
    color: black;        
}
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: #ffa500;
}</string>
    </property>
    <widget class="QPushButton" name="AddButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: #28a745;
    color: white;
    font-weight: bold; 
    border: 2px solid #218838; 
}
QPushButton:hover {
    background-color: #218838;
}
QPushButton:pressed {
    background-color: #1e7e34; 
}
</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: #ffa500;
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color: #28a745;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalExpensesLabel">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
